src/main/resources/Linux.xml

```diff
@@ -305,21 +305,26 @@
                     <cols>vmeff</cols>
                 </Plot>
             </Graph>
-            <Graph name="DEV" Title="Devices" type="multiple">
+            <Graph name="DEV_OLD" Title="Devices" type="multiple">
                 <Plot Title="tps">
                     <cols>tps</cols>
                 </Plot>
                 <Plot Title="blks/s">
                     <cols>blks/s</cols>
                 </Plot>
+            </Graph>
+            <Graph name="DEV" Title="Devices" type="multiple">
+                <Plot Title="tps">
+                    <cols>tps</cols>
+                </Plot>
                 <Plot Title="sector">
                     <cols>rd_sec/s wr_sec/s</cols>
                 </Plot>
                 <Plot Title="queue">
                     <cols>avgrq-sz avgqu-sz</cols>
                 </Plot>
-                <Plot Title="service">
-                    <cols>await svctm</cols>
+                <Plot Title="Response [ms]">
+                    <cols>await</cols>
                 </Plot>
                 <Plot Title="util">
                     <cols>%util</cols>
```

Comment:
Do you think this is important?
Frankly speaking, I would drop `%util`: https://brooker.co.za/blog/2014/07/04/iostat-pct.html

> As a measure of general IO busyness %util is fairly handy, but as an indication of how much the system is doing compared to what it can do, it's terrible. Iostat's svctm has even fewer redeeming strengths. It's just extremely misleading for most modern storage systems and workloads. Both of these fields are likely to mislead more than inform on modern SSD-based storage systems, and their use should be treated with extreme care.